model/textparse/benchmark_test.go

```diff
@@ -262,3 +262,77 @@ func readTestdataFile(tb testing.TB, file string) []byte {
 	require.NoError(tb, err)
 	return buf
 }
+
+/*
+	export bench=v3 && go test ./model/textparse/... \
+		 -run '^$' -bench '^BenchmarkCreatedTimestampPromProto' \
+		 -benchtime 2s -count 6 -cpu 2 -benchmem -timeout 999m \
+	 | tee ${bench}.txt
+*/
```

Comment:
Suggest this belongs in a doc file about running benchmarks; it is not special to this one benchmark.
(Also I don't know what the `bench=v3` does)